src/components/download-mushaf-button.tsx

```
@@ -0,0 +1,345 @@
+            alert(
+                'Service Worker is not active. Please ensure it is enabled and you are running on localhost or https.'
+            );
+            return;
+        }
+        swRef.current = sw;
+
+        updateOverallProgress();
+        setStatus('downloading');
+    };
+
+    const handlePause = () => {
```

ESLint

[error] Delete `··`

(prettier/prettier)